Bill3.java

```diff
@@ -1,8 +1,23 @@
-// Splits a restaurant bill evenly among three diners.
```

Comment:
The code contains inconsistent indentation, making it harder to read. Ensure proper formatting.